Update kaptain 2.0 version policy (#4377)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/version-policy/index.md

---

D2iQ supports `N-2` of the latest `MAJOR.MINOR` release of Kaptain.
With the GA release of Kaptain 2.0.0, support continues for Kaptain 1.3, and 1.2, including all patch releases. Support for Kaptain 1.1 expires.

D2iQ recommends that you install the newest Kaptain version available to stay up to date with the latest features and bug fixes.

## Supported DKP Versions

> Kaptain 2.0.0 runs on DKP 2.1.2 and newer and can be installed as a workspace catalog application.

Reviewer: ```suggestion
You can deploy Kaptain 2.0.0 on DKP 2.1.2 and later versions as a workspace catalog application.
```

Support for Experimental components is provided on a Standard level, Severity 4 basis only.

## Istio Support Status

Istio is a required prerequisite for Kaptain.
Istio usage is only supported in conjunction with Kaptain and the configurations delivered as part of Kaptain.
All other usage of [experimental]Istio[/experimental] is considered experimental.

## Secondary Support for Cassandra, Kafka & Spark

> <p class="message--note"><strong>Secondary Support Definition:</strong> Secondary support covers support for the base technology of platform service, which is Cassandra, Kafka, and Spark.</p>

Author: is this the correct html for this? I think it should be like "info" not red like this seems to be?

Reviewer: That would be:
```suggestion
<p class="message--note"><strong>NOTE: </strong><strong>Secondary Support Definition:</strong> Secondary support covers support for the base technology of platform service, which is Cassandra, Kafka, and Spark.</p>
```

Reviewer: Looks good to me in front-end. What do you mean with red?

Author: nothing.. the message is purple, not red 🙈
